frontend/pages/api/playground.ts

```diff
@@ -1,9 +1,17 @@
-if (process.env.NODE_ENV === 'development') require('nexus').default.reset()
+let handler
 
-const app = require('nexus').default
+if (process.env.NODE_ENV === 'development') {
+  require('nexus').default.reset()
+  require('../../nexus')
+  const app = require('nexus').default
 
-require('../../backend/src/graphql')
+  app.assemble()
+  handler = app.server.handlers.playground
+} else {
+  // Disable playground outside of development
+  handler = (_: any, res: any) => {
+    res.status(403).send('GQL playground is disabled outside of development')
+  }
+}
 
-app.assemble()
-
-export default app.server.handlers.playground
+export default handler
```

Comment:
Oooh so are we good with that question around configuring the playground page now!?

Reply:
Unfortunately we're not. This is just to disable the playground in prod environments once it works (users don't need it and the polling eats up a lot of requests).

Fixing it is probably a simple change in a prisma repo somewhere but IMO not a top priority since the electron playground app thing exists